app/src/main/kotlin/com/infinity/foodstandards/ui/authorities/AuthorisesAdapter.kt

```diff
@@ -0,0 +1,48 @@
+package com.infinity.foodstandards.ui.authorities
+
+import android.annotation.SuppressLint
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.infinity.foodstandards.R
+import com.infinity.foodstandards.databinding.LocalAuthorityItemBinding
+import com.infinity.foodstandards.model.LocalAuthority
+
+class AuthorisesAdapter(
+    private val onItemSelected: (LocalAuthority) -> Unit
+) : RecyclerView.Adapter<AuthorisesAdapter.ItemViewHolder>() {
```

Comment:
Probably worth changing this to a ListAdapter then we can remove `fun updateData` containing `notifyDataSetChanged()`